krabs/krabs/filtering/event_filter.hpp

```diff
@@ -49,6 +50,14 @@ namespace krabs {
          */
         event_filter(filter_predicate predicate);
 
+        /**
+         * <summary>
+         *   Constructs an event_filter that is setting trace native filters
+         *   events.
+         * </summary>
+         */
+		event_filter(unsigned short origEventId);
```

Comment:
nit: `origEventId` doesn't seem very descriptive. consider `providerFilterEventId`.

Reply:
Please use snake_case, not camelCase. So: `provider_filter_event_id`.

Reply:
Done